### ISSUE TITLE
[Metrics] Collect metrics for price estimation service

### ISSUE THREAD

**Issue by fleupold** [gnosis/dex-services#1274]
In order to measure reliability and performance of the price estimation service, we need to collect metrics such as uptime, http response time/status etc and visualize it in a dashboard similar to how we do it with the solvers.

**Comment by fleupold** [comment]
This is pretty much done, right? Last think missing is the dashboard in the production grafana.

**Comment by e00E** [comment]
Yes, we have metrics and dashboard in staging. There is some potential future work like adding another label to the per route response time metrics to differentiate the cached pricegraph (current orderobook) from uncached (past orderbook) because those requests take significantly longer to answer.
